_opensearch/query-dsl/geo-bounding-box.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: default
+title: Geo-bounding box queries
+parent: Query DSL
+nav_order: 55
+---
+
+# Geo-bounding box queries
+
+To search for documents that contain [geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point) fields, use a geo-bounding box query. The geo-bounding box query returns documents whose geopoints are within the bounding box specified in the query. A document with multiple geopoints matches the query if at least one geopoint is within the bounding box.
+
+## Example
+
+You can use a geo-bounding box query to search for documents that contain geopoints. 
+
+Create a mapping with the `point` field is mapped as `geo_point`:
+
+```json
+PUT testindex1
+{
+  "mappings": {
+    "properties": {
+      "point": {
+        "type": "geo_point"
+      }
+    }
+  }
+}
+```
+
+Index three geopoints as objects with latitudes and longitudes:
+
+```json
+PUT testindex1/_doc/1
+{
+  "point": { 
+    "lat": 74.00,
+    "lon": 40.71
+  }
+}
+
+PUT testindex1/_doc/2
+{
+  "point": { 
+    "lat": 72.64,
+    "lon": 22.62
+  } 
+}
+
+PUT testindex1/_doc/3
+{
+  "point": { 
+    "lat": 75.00,
+    "lon": 28.00
+  }
+}
+```
+
+Search for all documents and filter those documents whose points lie within the rectangle defined in the query:
+
+```json
+GET testindex1/_search
+{
+  "query": {
+    "bool": {
+      "must": {
+        "match_all": {}
+      },
+      "filter": {
+        "geo_bounding_box": {
+          "point": {
+            "top_left": {
+              "lat": 75,
+              "lon": 28
+            },
+            "bottom_right": {
+              "lat": 73,
+              "lon": 41
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The response contains the matching document:
+
+```json
+{
+  "took" : 20,
+  "timed_out" : false,
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "skipped" : 0,
+    "failed" : 0
+  },
+  "hits" : {
+    "total" : {
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "testindex1",
+        "_id" : "1",
+        "_score" : 1.0,
+        "_source" : {
+          "point" : {
+            "lat" : 74.0,
+            "lon" : 40.71
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+The preceding response does not include document 3 with a geopoint of `"lat": 75.00, "lon": 28.00` because of the geopoint's limited [precision](#precision).
```

Comment:
Do you think it's worthwhile adding "... include document 3 (`_doc/3`) with a geopoint of ..." ?
Or, even just say "document #3"
It may not be necessary. But it took me a second to understand what that referred to.

Reply:
I'll change to "the document with a geopoint...". The id is not too relevant.